spec.bs

```diff
@@ -7716,6 +7754,16 @@ The following algorithm will be added to the [[FETCH#fetching]] section:
       1. If |storedAuctionResultHeaders|[|requestOrigin|] [=map/exists=], then [=list/append=] |hash|
         to |storedAuctionResultHeaders|.
       1. Otherwise, [=map/set=] |storedAuctionResultHeaders|[|requestOrigin|] to « |hash| ».
+  1. Let |adAuctionResultNonces| be the result of [=header list/getting, decoding, and splitting=]

```

Comment:
`[=header list/Delete=] "[:Ad-Auction-Result-Nonce:]" from |responseHeaders|` after this step with a note similar to some other headers above, to match [code](https://crsrc.org/c/content/browser/interest_group/ad_auction_headers_util.cc;drc=e9545a72e6dadbfbff1b1c8ff5743bc90d2b15c4;l=355)